insertion_sort/Java/InsertionSort.java

```diff
@@ -0,0 +1,36 @@
+class InsertionSort {
+    /**
+     * Insertion sort proceduce. Sorts the array in increasing 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmptb1qj8_a/insertion_sort/Java/InsertionSort.java
+++ b/tmp/tmptb1qj8_a/insertion_sort/Java/InsertionSort.java
@@ -1,6 +1,6 @@
 class InsertionSort {
     /**
-     * Insertion sort proceduce. Sorts the array in increasing 
+     * Insertion sort proceduce. Sorts the array in increasing
      * order in place. Time - O(n2), space O(1)
      * @param arr the array to be sorted.
      */
```